kedro-datasets/README.md

```diff
@@ -17,6 +17,30 @@ Welcome to `kedro_datasets`, the home of Kedro's data connectors. Here you will
 pip install kedro-datasets
 ```
 
+### Install dependencies at a group-level
```

Comment:
```suggestion
### Install dependencies at a group level
```
Not sure why this is hyphenated, but I blamed the original source, and I it was added by @stichbury, so who am I to question the grammar. :joy:

Reply:
If @stichbury says it's hyphenated, it's hyphenated 🙇🏼

Reply:
Thanks! Although I'm not sure I'm right here, but there would have been a reason and it's consistent with the next heading, so let's keep it!